beetle_battle.py

```diff
@@ -5,30 +5,53 @@
 import arcade
 
 # Constants
-SCREEN_WIDTH = 1024
+SCREEN_WIDTH = 1280
 SCREEN_HEIGHT = 720
 SCREEN_TITLE = "Welcome to Arcade"
 
+# Scaling constants
+BEETLE_SCALING = .65
+
 # Classes
 class BeetleBattle(arcade.Window):
     def __init__(self):
 
         # Call the parent class constructor
         super().__init__(SCREEN_WIDTH, SCREEN_HEIGHT, SCREEN_TITLE)
         
+        # Beetle sprites in this variable
+        self.beetle_list = None
+        self.beetle_sprite = None
+
         # Background image stored in this variable
         self.background = None
 
         # Set the background window
         arcade.set_background_color(arcade.color.WHITE)
 
     def setup(self):
+        # Create the background
         self.background = arcade.load_texture("Assets/Images/octagon.png")
 
+        # Create sprite lists
+        self.beetle_list = arcade.SpriteList()
+
+        # Set up the beetle and place it at these coordinates
+        beetle_sprite_source = "Assets/Sprites/beetle_GRN.png"
```

Comment:
Since this value doesn't change, maybe we should make this a constant. I think I would also use `BEETLE_SPRITE_PATH_GREEN` or something like that (especially since we will eventually have `BEETLE_SPRITE_PATH_RED`). I just feel like "path" is a more common word for this type of string than "source", we need to go to upper-case if we're doing constants, and it's probably good to get ahead of the curve with the different colors.

With that in mind, I would also consider moving the definition of this variable out to where the other constants are. Where it is now, it is allocated and initialized on every call to `setup`. That being said, you can argue that it only makes sense here in `setup`. But to be honest, I bet we should eventually move this out to its own helper function like `load_beetle_sprite` or something. But we don't need to do that now.